wake-lock/wakelock-type.https.any.js

```diff
@@ -0,0 +1,19 @@
+//META: title=WakeLock.request() with invaild type
+
+promise_test(async t => {
+  await promise_rejects(t, new TypeError(), WakeLock.request());
+}, "'TypeError' is thrown when set an empty wake lock type");
+
+promise_test(async t => {
```

Comment:
You can drop `async` here since it's only needed in line 16.

Reply:
Ops... Done!